configs/gotc/cromwell.conf.ctmpl

```diff
@@ -0,0 +1,395 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{with $cromwellSecrets := vault (printf "secret/dsde/gotc/%s/cromwell/secrets" $environment)}}
+
+include required(classpath("application"))
+
+webservice {
+  port = 8001
+  interface = 0.0.0.0
+  instance.name = "reference"
+  binding-timeout = 30s
+}
+
+akka {
+  actor.default-dispatcher.fork-join-executor {
+    parallelism-factor = 100.0
+    parallelism-max = 64
+  }
+
+  #Increased from 20s default to allow large metadata responses
+  http.server.request-timeout = 55s
+  http.server.idle-timeout = 55s
+
+  # Don't warn in server logs (and thus spam Sentry) if client headers are malformed
+  http.server.parsing.illegal-header-warnings = off
+}
+
+system {
+  # If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
+  #abort-jobs-on-terminate = false
+
+  # If 'true', a SIGTERM or SIGINT will trigger Cromwell to attempt to gracefully shutdown in server mode,
+  # in particular clearing up all queued database writes before letting the JVM shut down.
+  # The shutdown is a multi-phase process, each phase having its own configurable timeout. See the Dev Wiki for more details.
+  # GOTC NOTE: reenable this when we want to actually do graceful shutdowns (not just kill the container)
+  graceful-server-shutdown = false
+
+  # Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
+  max-retries = 10
+
+  # If 'true' then when Cromwell starts up, it tries to restart incomplete workflows
+  workflow-restart = true
+
+  # Cromwell will cap the number of running workflows at N
+  max-concurrent-workflows = 12500
+
+  # Cromwell will launch up to N submitted workflows at a time, regardless of how many open workflow slots exist
+  max-workflow-launch-count = 30
+
+  # Number of seconds between workflow launches
+  new-workflow-poll-rate = 10
+
+  # Lowered from the default number of cache read workers (25)
+  number-of-cache-read-workers = 10
```

Comment:
ToL dupe of 2 settings below, I'd recommend deleting both since one of them is the default and it's not clear why it was lowered in the first place